File: moviejava/app/src/main/java/com/example/moviejava/data/local/AppDatabase.java
package com.example.moviejava.data.local;

import androidx.room.Database;
import androidx.room.RoomDatabase;
;import com.example.moviejava.data.local.dao.AppDao;
import com.example.moviejava.data.local.entities.MovieEntity;

@Database(entities = {MovieEntity.class}, version = 1, exportSchema = false)
public abstract class AppDatabase extends RoomDatabase {
  public   abstract AppDao appDao();
}

File: movie/app/src/main/java/com/example/movie/data/local/AppDatabase.kt
package com.example.movie.data.local

import androidx.room.Database
import androidx.room.RoomDatabase
import com.example.movie.data.local.dao.AppDao
import com.example.movie.data.local.entities.MovieEntity

@Database(entities = [MovieEntity::class], version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {
    abstract fun appDao(): AppDao
}